### ISSUE TITLE
Request: Quickly toggle DNS server

### ISSUE THREAD

**Issue by greenpasta** [boxeehacks/boxeehack#184]
It would be great if there was some way to quickly toggle back and forth between DNS servers.

**Comment by dmike3** [comment]
Update 1.4 will include a quick toggle of VPN.

**Comment by greenpasta** [comment]
I don't mean the VPN connection, I mean the actual DNS server you use in your network settings.  I don't think it's the same thing.  

Services like unblock-us.com just have you change your DNS server to access U.S. content for certain services (e.g. Netflix).  There are also similar free services out there, but those aren't very reliable.  Normally I'll just watch my country's local Netflix, so I just use my regular DNS server.  But if there is a particular show/movie I am looking to watch that is only on U.S. Netflix, I'll temporarily change my DNS server.  I don't want to leave my DNS server changed all the time because things can get wonky, especially with the free ones. 

As it is now, it isn't very convenient to manually go look up and type in a different DNS server each time I want to do this.

**Comment by Alassalaur** [comment]
I agree with this, a way to quickly switch DNS would be extremely useful. Right now, to switch manually you need to type in a DNS every single time you want a DNS that doesn't pull from your router's info. Really, a way to save more than one DNS so they can they be switched between is what is needed.

**Comment by tekkietekkie** [comment]
Agree this would be a good feature

**Comment by Balwyn** [comment]
Amen to this. If a name could be assigned to each stored DNS address and the home page had a means of selecting DNS setting by name, this would be ideal for me. You could set the machine to "US Mode" , "UK Mode" "Australia Mode" etc.
